lib/index.js

```diff
@@ -194,6 +194,11 @@ class BundleTrackerPlugin {
 
       output.assets[assetName] = fileInfo;
     });
+    each(Object.fromEntries(stats.compilation.assetsInfo), ({ sourceFilename }, assetName) => {
+      if (!sourceFilename) return;
+
+      output.assets[assetName].sourceFilename = sourceFilename;
+    });
```

Comment:
I think you can move this logic to be fileInfo, like: `fileInfo.sourceFilename = file.sourceFilename;`

This will prevent this second each.